decidim-elections/spec/commands/decidim/votings/census/admin/create_dataset_spec.rb

```diff
@@ -32,6 +32,16 @@ module Decidim::Votings::Census::Admin
       expect(subject.call).to broadcast(:ok)
     end
 
+    # rubocop:disable Lint/AmbiguousBlockAssociation, RSpec/IteratedExpectation
+    it "enqueues a job for processing the dataset and strips the data from whitespaces" do
+      expect { subject.call }.to have_enqueued_job(CreateDatumJob).at_least(4).times.with { |_user, _dataset, row|
```

Comment:
⚠️  You should use `do` instead of `{` for the `times.with` block, because it's a multiline one.